Tools/WebKitTestRunner/TestController.h

```diff
@@ -533,7 +534,7 @@ class TestController {
     bool downloadDidReceiveServerRedirectToURL(WKDownloadRef, WKURLRequestRef);
     static void downloadDidReceiveAuthenticationChallenge(WKDownloadRef, WKAuthenticationChallengeRef, const void *clientInfo);
 
-    void downloadDidWriteData(long long totalBytesWritten);
+    void downloadDidWriteData(long long totalBytesWritten, long long totalBytesExpectedToWrite);
```

Comment:
These should probably all be uint64_t instead of long long, but expected bytes should be std::optional<uint64_t>